content/topics/Analyze/machine-learning/ml-intro/supervised-learning-101.md

```diff
@@ -0,0 +1,333 @@
+It predicts the output based on the majority vote (for classification) or the average (for regression) of the "k" number of nearest neighbors.
+
+{{% warning %}}
+It is important to select the `k` that suits your data. A small value of `k` means that noise will have a higher influence on the result, whereas a large value makes the algorithm slower and may also make it too general. The best value of `k` is usually found through cross-validation.
+{{% /warning %}}
+
+- **Decision Trees**: 
+
+Decision trees can be used for both classification and regression, and is often used because it is a visually interpretable model. 
+It builds a tree where each so-called "node" splits the data based on a simple decision rule. Each "branch" then splits the data on the outcome of this rule. 
+The deeper the tree, the more complex these decision rules get. Below is an example of a decision tree that classifies flowers in one of three species with four input features.
+
+<p align = "center">
+<img src = "/images/decision-tree-iris.png" width="700">
+<figcaption> Visualization of a decision tree, image by the author </figcaption>
+</p>
+
+{{% tip %}}
+Decision trees are great for visual interpretability. However, due to their high variance, they tend to perform worse in terms of predictability. 
+To reduce this variance and to make better predictions, you can use Random Forests.
+{{% /tip %}}
+
+- **Random Forests**: 
+
+Random forests is an ensemble method that builds multiple decision trees to improve prediction accuracy and control overfitting. 
+Random forests average many decision trees to reduce their inherent high variance. Read more in [this Introduction to Random Forests in Python](/randomforest).
+
+- **Support Vector Machines (SVM)**
+
+SVM is a powerful algorithm for classification problems. It identifies a line called a "hyperplane" that separates different categories. The algorithm's objective is to place this hyperplane in such a way that the gap between the closest points of different classes on either side of the line is maximized. An example of SVM is face detection, where the algorithm classifies parts of an image as a face or non-face and recognizes different facial expressions.
+
+- **Neural Networks**
+
+Neural Networks are inspired by the structure and function of the brain's neural networks. They consist of layers of nodes, or "neurons," that make simple calculations from their input. The data goes through an input layer, then through one or more "hidden" layers, and finally through an output layer. Neural networks are particularly effective for complex problems. An example is image recognition, where a neural network can identify and classify objects within images.
+
+{{% warning %}}
+Neural networks can model complex patterns, but they require large amounts of data and computational power. 
+Moreover, they act as "black boxes," making it difficult to understand the exact nature of the decision-making processes.
+{{% /warning %}}
```

Comment:
If you want to put this as a warning, I'd rather emphasize the 'black box' idea at the beginning: warning readers about that choosing a neural network over other algorithms can complicate the understanding of the decision making process. Otherwise to me it looks like is some extra information that could be out of the warning box.

Moreover, I would highlight in the warning that it can be also very computationally intensive to train a neural network while some algorithms can be way more efficient while achieving similar results. So this should be considered depending on the problem domain.